reference/concepts.md

```diff
@@ -66,19 +66,19 @@ The Haskell concept exercises are based on concepts. The list below contains the
 - [ ] Resources
   - [ ] Resource allocation
 - [ ] Scoping
-  - [ ] Imports
+  - [x] Imports
```

Comment:
I can't agree that these have been covered. I would expect some bit of explanatory text about `import`, but none is to be found.

Reply:
You are correct. I thought I had touched on imports in `basics` but it was only module definition and function exports.